### PR TITLE
QA Update installation.md

### PR DESCRIPTION
**Pull request by qahmed1998**

### CODE REVIEW

**Comment by qahmed1998** [comment]
@miq-bot assign @Fryguy

**Comment by qahmed1998** [comment]
@miq-bot add_labels radjabov/yes

File: installing_on_google_compute_engine/_topics/installation.md [hunk 1]
```diff
@@ -9,35 +9,35 @@ Installing {{ site.data.product.title_short }} consists of the following steps:
...
+- 44 GB of storage space
+- 12 GB RAM
+- 4 vCPUs.
 
 You can upload the appliance with the following steps:
 
 1.  Log in to the Google Cloud Platform dashboard.
 
-2.  Click **Home** in the top left of the screen.
+2.  Click **Home** located on the left of the screen.
```
Comment (bdunne) [inline installing_on_google_compute_engine/_topics/installation.md:31]:
```suggestion
2.  Click **Home** located in the top left corner of the screen.
```

File: installing_on_google_compute_engine/_topics/installation.md [hunk 2]
```diff
@@ -9,35 +9,35 @@ Installing {{ site.data.product.title_short }} consists of the following steps:
...
 3.  Click ![gce products services](../images/gce-products-services.png) to
     show the **Products and Services** menu. Click **Storage**.
 
 4.  Create a bucket by clicking **Create Bucket**, and configure the
     following details:
 
-    1.  Enter a unique **Name** for the bucket using lower case
-        alphanumeric characters, hyphens, and/or underscores.
+    1.  Enter a unique **Name** for the bucket that uses lowercase
+        alphanumeric characters, hyphens, and underscores.
```
Comment (bdunne) [inline installing_on_google_compute_engine/_topics/installation.md:40]:
```suggestion
    1.  Enter a unique **Name** for the bucket.  Valid characters include lowercase alphanumeric characters, hyphens, and underscores.
```

File: installing_on_google_compute_engine/_topics/installation.md [hunk 3]
```diff
@@ -53,50 +53,47 @@ You can upload the appliance with the following steps:
     services > Compute Engine** on the left menu.
 
 7.  Create an image by clicking **Images** from the left menu, then
-    **Create Image**. Fill in the following details about the image:
+    **Create Image**. Enter the following details about the image:
 
-    1.  Enter a unique **Name** for the image using lower case
-        alphanumeric characters and/or hyphens.
+    1.  Enter a unique **Name** for the image that uses lowercase
```
Comment (bdunne) [inline installing_on_google_compute_engine/_topics/installation.md:58]: Same comment as above

File: installing_on_google_compute_engine/_topics/installation.md [hunk 4]
```diff
@@ -53,50 +53,47 @@ You can upload the appliance with the following steps:
...
+        alphanumeric characters and hyphens.
 
-    2.  Add a **Description** if desired.
+    2.  Add a **Description**.
```
Comment (bdunne) [inline installing_on_google_compute_engine/_topics/installation.md:61]:
```suggestion
    2.  Optionally, add a **Description**.
```

File: installing_on_google_compute_engine/_topics/installation.md [hunk 5]
```diff
@@ -53,50 +53,47 @@ You can upload the appliance with the following steps:
...
+    6.  Click **Create**. Creating an image can take a few minutes.
+        When the image is created, the screen refreshes, and the new
+        image appears in the **Images** list.
 
 8.  Create a virtual machine instance by navigating to **VM
     instances > Create Instance**, and configure the following fields:
 
-    1.  Enter a unique **Name** for the virtual machine instance using
-        lower case alphanumeric characters and/or hyphens.
+    1.  Enter a unique **Name** for the virtual machine instance that uses
```
Comment (bdunne) [inline installing_on_google_compute_engine/_topics/installation.md:80]: Same comment as above

File: installing_on_google_compute_engine/_topics/installation.md (hunk 1 above)
Comment (qahmed1998) [reply installing_on_google_compute_engine/_topics/installation.md:31]: We can't say things like top left bottom right etc. These introduce accessibility violations.

**Comment by qahmed1998** [comment]
Hi @bdunne I have pushed the minor fixes.

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit bbf14546b6c4a2ff26cbad1b822450d994c4f0e0.

```text
commit bbf14546b6c4a2ff26cbad1b822450d994c4f0e0
Author: Jason Frey <fryguy9@gmail.com>
Date:   Thu Jul 11 15:55:04 2024 -0400

    Merge pull request #1795 from qahmed1998/QA-install-on-gcp
    
    QA Update installation.md
    
    (cherry picked from commit e8a07b9112fc71e6a8537a3f8a19680c5ee9a431)
```

**Pull request merged**